bin/main.ml

```diff
@@ -0,0 +1,7 @@
+(** Copyright 2021-2024, MacadamiaSolver. *)
+
+(** SPDX-License-Identifier: MIT *)
+
+open Lib
+
+let () = print_endline (HelloWorld.getString ())
```

Comment:
Probably it's better to move these files into an `src/` because the `bin` directory historically associates with something executable.

Though I don't insist because it may be the template.

Reply:
It is in fact the template for projects in dune and quite popular in OCaml projects so I think we can leave this the way it is now